### ISSUE TITLE
A11y_AzureNotebooks_Jupyter_FilesTab_Inspect:Role is incorrect for the "New" dropdown..

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [jupyter/notebook#3973]
**EnvironmentDetails:**
Application Name:AzureNotebooks_Jyupter
Login Url: notebooks.azure.com
BrowserDetails:
Edge:Microsoft Edge 41.16299.611.0
Chrome:Version 68.0.3440.106 
Operating System:Windows10
 
 **Steps to reproduce:**
1.Login with the appropriate credentials to azure notebook account.    
2.Go to Get started à libraries à Select any of the libraries à click on clone to clone the library
3.Click on Run or right click and select Run to run the notebook.
5.Navigate to the "New" control.
6.Start the Narrator ot inspect.
**Actual:**
Role is incorrect for the "New" contol
**Expected:**
Correct role for the "New" should be dropdown, as it has list items and sub menu list-items associated to it.
 **UserImpact:**
User will not get correct information regarding the control type of the Drop-down
**Attachment:**
![attachment1](https://user-images.githubusercontent.com/39204909/45803385-1f110600-bcd7-11e8-9d98-9a02ef2e8f8f.jpg)

**Comment by joshuazeltser** [comment]
I have been attempting to solve this problem and have been able to get the narrator to now say "New Toggle Dropdown, expanded, button". This makes what this button is a bit clearer. Does this seem clear enough?

**Comment by takluyver** [comment]
Closed by #4691.
